tests/acls/acl.sh.in

```diff
@@ -15,6 +15,8 @@ change_file(){
     sed "s/$1/$2/g" netdata.cfg > "$4"
 }
 
+NETDATAPID=123456789
```

Comment:
There was a PR where @mfundul said me that to use `killall` command would not be good, because it could send unecessary commands, no less important I verified that the `killall` has not been installed more in some distributions, so instead to start  `netdata` as a daemon and to use `killall` I am staring it with the parameter `-D` and getting the PID. 
The value that you are seeing, it is an arbitrary value that I am using to do not start a variable without any value. This variable is visible in different parts of the code.

Reply:
Garbage value is worse than no value. So just set it when you get the PID and don't use it before you do.

Reply:
All right, I understood. I removed it with the last commit.